### DISCUSSION TITLE
Anbindung mit USB2Serial möglich ?

### DISCUSSION THREAD

**Discussion by gittysj** [cbrosius/luxtronik_v1#27] [Q&A]
Hallo zusammen,

ich habe auch noch eine alte Alpha Innotec Wärmepumpe und wollte diese auch in mein home assistant integrieren.
Jetzt bin auf Eure Anleitung gestossen :-)
Echt super was Ihr da zur Verfügung stellt!

Mit meiner "Unwissenheit" wollte ich mal fragen, ob man nicht auch einen USB2Serial Adapter nutzen kann um die Wärmepumpe abfragen ?
Das wäre weniger "aufwendig" als den Weg über das ESP HW+Modul... Oder sehe ich das als Laie zu unwissend ?

Zur Zeit nutze ich schon erfolgreich das Perl Script von hoerndlein mit einem USB2Serial Adapter und Speicherung auf einem raspberry.
(Ist aber nicht so schick wie homeassistant und erfordert ein Pi-Zusatzgerät)

Vielen Dank schonmal
Stephan

**Comment by cbrosius** [comment]
Hallo Stephan,

selbstverständlich kannst Du auch einen USB2Serial Adapter verwenden. Aber wo schließt Du den an?
Dann benötigst Du einen Rechner in der Nähe der Wärmepumpe, wie Deinen PI mit dem Perl-Script.

Der ESP hat den Vorteil, daß Du keinen Raspberry PI oder anderen Rechner in der Nähe benötigst. Du brauchst nur WLAN-Empfang.
Die abgefragten Daten, stellt er dann direkt Homeassistant, oder optional über MQTT beliebigen anderen SmartHome-Systemen zur Verfügung.

Gruß,
Christian...

**Reply by gittysj** [reply]
Hi Christian,

ich dachte einfach an den HomeAssistant Host (das wäre bei mir ein raspberry).
Dazu habe ich mal gesucht aber nichts gefunden was mir weiterhilft - lediglich mit Bordmitteln sehe ich den Stick im Linux.


VIele Grüße
Stephan
